EarlGrey/Additions/CALayer+GREYAdditions.m

```diff
@@ -73,6 +73,11 @@ - (void)grey_adjustAnimationToAllowableRange:(CAAnimation *)animation {
     return;
   }
 
+  CFTimeInterval layerSpeed =
```

Comment:
Let's talk about this on Slack. I might have jumped the gun by suggesting this too early. Ping me whenever you're available.